stuff/broken.jsx

```diff
@@ -0,0 +1,6 @@
+a = 1
+
+
+5 + 5
```

Comment:
_[farcy v0.1.1](https://github.com/appfolio/farcy)_
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.